### PR TITLE
closes #12

### LINKED ISSUE #12: option to add new issue comment rather than updating existing one
The current action run on existing issues only calls `issues.update` which updates the latest comment in an issue.
https://github.com/ropensci-review-tools/pkgcheck-action/blob/b5d9c7858a13cdc3b3cdb794efbc81f1054e1dd1/action.yaml#L86-L92
Could easily add an additional option to instead call [`issue.createComment`](https://octokit.github.io/rest.js/v18#issues-create-comment) so each run would add another comment to the same thread. @assignUser Would it be okay if I try doing that, to give me a change to actively edit the action here, and help me understand it better? If you're okay with that, then please assign me, and I'll start a PR and ask for your advice there.

### PR DESCRIPTION
**Pull request by mpadge**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -96,6 +100,14 @@ runs:
                   body: issue_body,
               })
           } else if (${{ steps.get-id.outputs.result == -1 && inputs.post-to-issue }}) {
+            if (${{ inputs.append-to-issue }}) {
```
Comment (assignUser) [inline action.yaml:103]: This needs to be in the if block when `result != -1` otherwise there is no issue to comment on. Looks good otherwise!

Comment (mpadge) [reply action.yaml:96]: Ah yeah, thanks, all fixed now.

File: action.yaml [hunk 2]
```diff
@@ -89,12 +93,21 @@ runs:
           const repo = "${{ steps.get-repo.outputs.result }}"
 
           if (${{ steps.get-id.outputs.result != -1 && inputs.post-to-issue }}) {
+            if (${{ inputs.append-to-issue }}) {
+              github.rest.issues.createComment({
+                owner: owner,
+                repo: repo,
+                issue_number: "${{ steps.get-id.outputs.results }}",
+                boxy: issue_body
```
Comment (assignUser) [inline action.yaml:101]:
```suggestion
                body: issue_body
```
:D https://github.com/assignUser/simstudy/runs/5006623052?check_suite_focus=true#step:3:1267

**Review by assignUser — CHANGES_REQUESTED** [review]

**Comment by assignUser** [comment]
This is still not working, not sure why: https://github.com/assignUser/simstudy/runs/5007245762?check_suite_focus=true#step:3:1266

**Review by assignUser — CHANGES_REQUESTED** [review]

**Pull request merged**